GI05MOAS-PG565/GI05MOAS-PG565_D00002_ingest.csv

```diff
@@ -0,0 +1,8 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-01-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-03-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-04-FLORTO000,telemetered,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-05-NUTNRM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-06-PARADM000,telemetered,Available,
```

Comment:
looks good

Reply:
Looks good.  But there is not data in the Raw Data Repo.